pkg/apis/storage/util/helpers.go

```diff
@@ -89,6 +91,15 @@ func GetStorageClassAnnotation(obj api.ObjectMeta) string {
 	return ""
 }
 
+// GetStorageClassProjects
+func GetStorageClassNamespaces(obj api.ObjectMeta) string {
+	if class, ok := obj.Annotations[StorageClassNamespaces]; ok {
```

Comment:
I don't care for this helper. It will not make sense when class graduates from annotation to field.